composer.json

```diff
@@ -44,7 +44,8 @@
         "league/csv": "~9.1",
         "nesbot/carbon": "^2.0",
         "laravel/framework": "~6.0",
-        "laravel/tinker": "~2.0"
+        "laravel/tinker": "~2.0",
+        "enshrined/svg-sanitize": "^0.14.1"
```

Comment:
Why so restrictive on the version? I think they're on 0.15 now

Reply:
I think that was inserted by `composer require`, I'll fix it up.